scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -415,8 +475,8 @@ Function RunQuickstartCertsTest
     TestSetup
 
     $testStartAt = Get-Date
-    $deviceId = "e2e-${ReleaseLabel}-Windows-${Architecture}-QuickstartCerts"
-    PrintHighlightedMessage "Run quickstart test with -d ""$deviceId"" started at $testStartAt"
+    $deviceId = "e2e-${ReleaseLabel}-Windows-${Architecture}-QuickstartCerts-$(Get-Date -UFormat %s)"
+    PrintHighlightedMessage "Run quickstart test with -d ""$deviceId"" and deployment file $DeploymentWorkingFilePath started at $testStartAt"
```

Comment:
I am ok to print it out, but $DeploymentWorkingFilePath will be the same for all tests.

Reply:
Yeah that was useful to me at some point during development but I don't remember why anymore. I'll back it out.